src/onepasswordconnectsdk/__init__.py

```diff
@@ -4,12 +4,5 @@
 from onepasswordconnectsdk.config import load_dict
 from onepasswordconnectsdk.client import new_client
 from onepasswordconnectsdk.client import new_client_from_environment
-
-__all__ = [
-    "client",
-    "load",
-    "load_dict",
-    "models",
-    "new_client",
-    "new_client_from_environment"
-]
```

Comment:
Won't deleting this block avoid these components from being properly exported from this package?